packages/code-studio/src/settings/ColumnSpecificSectionContent.tsx

```diff
@@ -399,8 +429,9 @@ export class ColumnSpecificSectionContent extends PureComponent<
       this.handleFormatRuleChange(i, 'columnName', e.target.value);
     const onNameBlur = (): void =>
       this.handleFormatRuleChange(i, 'isNewRule', false);
-    const onTypeChange = (e: ChangeEvent<HTMLSelectElement>): void =>
+    const onTypeChange = (e: ChangeEvent<HTMLSelectElement>): void => {
       this.handleFormatRuleChange(i, 'columnType', e.target.value);
+    };
```

Comment:
Not sure why you added these here - `handleFormatRuleChange` doesn't return anything, so you can keep the shorthand. Or update the `onNameChange` and `onNameBlur` declarations to match for consistency.